Sources/AblyAssetTrackingCore/Model/EnhancedLocationUpdate.swift

```diff
@@ -0,0 +1,24 @@
+import CoreLocation
+
+/**
+ Enumeration used to determine enhanced location type.
+ */
+public enum LocationUpdateType: String, Codable {
+    case predicted = "PREDICTED"
+    case actual = "ACTUAL"
```

Comment:
same as before: until this is for internal use, it's not a good practice to assign a string name to string type enum, especially uppercased. You can leave it as it is if an uppercased name is required by an external API
```suggestion
    case predicted
    case actual
```

Reply:
Same comment as above 😅